src/utils.py

```diff
@@ -59,3 +69,63 @@ def get_repo_approvers():
         data = yaml.load(fd.read(), Loader=yaml.SafeLoader)
 
     return data["approvers"]
+
+
+def set_commit_status_success_semver(commit):
```

Comment:
How about creating generic set_commit_status_success and set_commit_status_pending functions and pass the description and context?

Reply:
great idea, but in different PR please.